pkg/cluster/info.go

```diff
@@ -28,17 +28,20 @@ import (
 	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
 )
 
 type Info struct {
 	Name           string
+	RestConfig     *rest.Config
 	ClientProducer client.Producer
 	Submariner     *v1alpha1.Submariner
 }
 
-func NewInfo(clusterName string, clientProducer client.Producer) (*Info, error) {
+func NewInfo(clusterName string, clientProducer client.Producer, config *rest.Config) (*Info, error) {
 	info := &Info{
 		Name:           clusterName,
+		RestConfig:     config,
```

Comment:
`RestConfig` is needed by `gatherCNIResources` -> `execCmdInBash` -> `pods.ExecWithOprions` leading to https://github.com/submariner-io/submariner-operator/blob/devel/internal/pods/exec.go#L90